quinn-h3/Cargo.toml

```diff
@@ -30,8 +30,8 @@ futures = "0.3.1"
 futures-util = { version = "0.3", default-features = false }
 http = "0.2"
 lazy_static = "1"
-quinn-proto = { path = "../quinn-proto", version = "0.5.0" }
-quinn = { path = "../quinn", version = "0.5.0" }
+quinn-proto = { path = "../quinn-proto" }
```

Comment:
Why do this? We'll need a specific version whenever we start publishing `quinn-h3`.

Reply:
Yeah that was not useful at all. Now I remember `0.5.0` matches `0.5.X`.

Reply:
I removed the commit